### DISCUSSION TITLE
[VOTE] Release Apollo version 2.0.0

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4348] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo version 2.0.0.

Release Note:

 * https://github.com/apolloconfig/apollo/releases/tag/v2.0.0

Release Candidate:

 * https://github.com/apolloconfig/apollo/releases/download/v2.0.0/apollo-configservice-2.0.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.0.0/apollo-adminservice-2.0.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.0.0/apollo-portal-2.0.0-github.zip
 * sha1 checksums
   - 11de373fc37d2b7529dad5828284ff2a67884636  apollo-configservice-2.0.0-github.zip
   - e81872916d5e05ca01aa5f4980e672c96a207367  apollo-adminservice-2.0.0-github.zip
   - f8e495b95e9cd332ff61e5734f3712f82714489f  apollo-portal-2.0.0-github.zip

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1040/content/com/ctrip/framework/apollo/

```xml
<repositories>
    <repository>
        <id>sonatype-nexus-release</id>
        <url>https://oss.sonatype.org/service/local/repositories/comctrip-1040/content</url>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

Release Tag :

 * https://github.com/apolloconfig/apollo/releases/tag/v2.0.0

Release CommitID :

 * https://github.com/apolloconfig/apollo/commit/cb308cb7a9b08178046f747c1f02c8385eeca28c

Guide to build the release from source:

 * https://www.apolloconfig.com/#/en/deployment/distributed-deployment-guide?id=_2212-building-from-source-code

Voting will start now (2022/5/12) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by zouyx** [comment]
+1 binding to release

I've checked 
- [x] All uts passed.
- [x] Release Note is fine.
- [x] Download links are fine.
- [x] Shasum are correct.

**Comment by Anilople** [comment]
+1 binding to release

I've checked 
- [x] All uts passed.
- [x] Download links are fine.
- [x] Shasum are correct.

**Comment by nisiyong** [comment]
+1 binding to release, and I have Checked
- [x] All uts passed.
- [x]  Release Note.
- [x]  Shasum are correct.
- [x] Download links are fine.
- [x] Deploying the new releases are fine.

**Comment by nobodyiam** [comment]
The vote to release Apollo version 2.0.0 has passed with three +1 binding and zero -1.
